contrib/mesos/pkg/node/node.go

```diff
@@ -55,6 +55,17 @@ func Create(
 		},
 		Status: api.NodeStatus{
 			Phase: api.NodePending,
+			// WORKAROUND(sttts): make sure that the Ready condition is the
```

Comment:
We can wait until a new kube-ui is out and the fix merged. But we will move this into the controller manager soon anyway, then this workaround goes away but itself. Moreover, having this behavior (without the kube-ui comment) would be completely reasonable even without this very reason.